engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/TypeDefinitionSet.scala

```diff
@@ -0,0 +1,44 @@
+package pl.touk.nussknacker.engine
+
+import org.springframework.expression.spel.SpelNode
+import pl.touk.nussknacker.engine.definition.{DefinitionExtractor, ProcessDefinitionExtractor, TypeInfos}
+
+
+case class TypeDefinitionSet(typeDefinitions: Set[TypeInfos.ClazzDefinition] = Set()) {
+
+  def validateTypeReference(spelNode: SpelNode) : Boolean = {
```

Comment:
It's scala - try to not use return :)

Reply:
I understand but here I don't see how I could avoid it, other then by using a flag which I would return at the end, and that would create a redundancy by searching the whole set every time even after it finds a match.